sdk/dotnet/ElasticSearch/GetDomain.cs

```diff
@@ -141,7 +141,7 @@ public sealed class GetDomainResult
         /// Status of a configuration change in the domain.
         /// * `snapshot_options` – Domain snapshot related options.
         /// </summary>
-        public readonly string Processing;
+        public readonly bool Processing;
```

Comment:
This is breaking too

Reply:
This is type-breaking, but did the API work at all previously?  My guess is no. That the actual value that was returned the provider was a bool already?

That aspect of “was there a working program that no longer works” is a key ingredient in the breaking change policy proposal I am drafting. 

Another point is “does it impact <1% of users”. This feels definitely in that category.